### DISCUSSION TITLE
discussion/

### DISCUSSION THREAD

**Discussion by giscus** [new-epilepsy/new-epilepsy#24] [General]
# discussion/

Discussion board to discuss things that needs discussing.

http://127.0.0.1:4000/discussion/

**Comment by new-epilepsy** [comment]
### 📣 Hello! 

We’re using GitHub Discussions as a place to connect with other members of NEW. We hope that you:

- Ask questions you’re wondering about.
- Promote your work.
- Share ideas, interesting articles, job ads, talks, etc.
- Engage and collaborate with other NEW members.
- Welcome others and are open-minded.

Don't be shy ✨

**Comment by new-epilepsy** [comment]
#### JOB OPPORTUNITIES 🧰

Are you passionate about advancing the field of computational neurology? [Newcastle University](https://www.ncl.ac.uk/) is seeking talented individuals to join their dynamic team. They have several exciting opportunities currently open and on the horizon! 

Find out more [here »](https://new-epilepsy.com/new/2024.01.10/)

**Comment by royj23** [comment]
# MRI basics in epilepsy: Data acquisition and organization

So, you're interested in neuroimaging of epilepsy? 👀 This post will give you some helpful tips and tricks to get started! 

## Data acquisition: Where it all begins

### Clinical guidelines

Clinical guidelines have been established regarding the type of sequences and their minimum resolution that should be acquired for epilepsy diagnosis and care. These guidelines are described in the [Harmonized Neuroimaging of Epilepsy Structural Sequences - or HARNESS-MRI - protocol](https://onlinelibrary.wiley.com/doi/full/10.1111/epi.15612). HARNESS-MRI also discusses contexts for first and repeated patient referrals for MRI, expected signal changes associated with different pathologies, and image post-processing strategies.

### Acquisition protocols

If you would like to put together your own epilepsy neuroimaging study, feel free to have a look at detailed protocols implemented in other centres [here](https://drive.google.com/drive/folders/1FlnR2Q2Y4w1AMXkxafuggLbx7ffysx6J?usp=drive_link).

Sharing is caring: If you would like to contribute a protocol from your centre to this list, please get in touch: *info[at]new-epilepsy[dot]com* - We're particularly interested in receiveing protocols designed for different scanner manufacturers! 🧠

## How to organize your data

Once you've acquired all of this beautiful data, how should you organize it? The neuroimaging community has come up with a standard for naming, organizing, and sharing data obtained in neuroimaging experiments, called the [Brain Imaging Data Structure or BIDS](https://bids.neuroimaging.io/). BIDS was initially developped for [MRI](https://www.nature.com/articles/sdata201644), but extensions for many different data modalities have been published in recent years. 

Although making your dataset BIDS-conform might seem like a lot of effort, it opens a world of opportunity for data analysis. Numerous software tools for MRI data pre- and post-processing, most notably [BIDS Apps](https://bids-apps.neuroimaging.io/), will simply not work on datasets that are not organized according to BIDS. NEW highly encourages the adoption of BIDS in the neuroimaging of epilepsy community, as they facilitate data sharing between groups and the development of reproducible and generalizable neuroinformatics pipelines. 

## Keep the discussion going!

Are you aware of any other standards in selecting, acquiring, and organizing imaging data in epilepsy? Drop a comment below 👇

**Reply by saratheriver** [reply]
###    GE Discovery MR750 3.0 Tesla MRI scanner 🧲

I'm also sharing our (research) acquisition protocol for pediatric epilepsy. You can find it [here](https://github.com/new-epilepsy/website-extras/blob/main/imaging-protocols/BCCH-Pediatric-MRI-Protocol.pdf)!

**Reply by drzeke** [reply]
I have shared the protocol we get on the 3T Siemens Skyra with DTI/DKI. All feedback welcome on potential improvements!

**Reply by royj23** [reply]
Thanks Sara and Zeke! Both protocols have been added to the shared Drive :point_right: [here](https://drive.google.com/drive/folders/1FlnR2Q2Y4w1AMXkxafuggLbx7ffysx6J?usp=drive_link) !

**Comment by royj23** [comment]
# Open neuroimaging datasets in epilepsy

Looking for open epilepsy datasets? Check out these resources!

## ENIGMA-Epilpesy

ENIGMA-Epilepsy is dedicated to improving our understanding of in vivo neuroanatomical disruptions in people with epilepsy compared to healthy individuals in the general population. It aggregates neuroimaging data from centres around the world and provides post-processed brain metrics alongside socio-demographic and clinical information. Data modalities include T1-weighted (T1w), diffusion-weighted (DWI), and resting-state functional MRI (rs-fMRI) collected in diverse patient populations (focal epilpesies, temporal and extratemporal, as well as generalized epilepsy syndromes). 

[Details here.](https://enigma.ini.usc.edu/ongoing/enigma-epilepsy/)

#### Shortcut
Summary statistics tables for ENIGMA-Epilepsy (and other disorders) are openly available as part of the [ENIGMA toolbox.](https://enigma-toolbox.readthedocs.io/en/latest/pages/04.loadsumstats/index.html#epilepsy)

## Epilepsy Connectome Project

The Epilepsy Connectome Project (ECP) will focus on gaining a better understanding of why some patients respond well to anticonvulsant therapy, whereas others develop controlled seizures and progressive brain dysfunction, with a focus on TLE.

[Details here.](https://www.humanconnectome.org/study/epilepsy-connectome-project)

## Australian Epilepsy Project

The Australian Epilepsy Project (AEP) is a large-scale clinical research project. The vision of the AEP is to develop predictive epilepsy-specific decision support tools for use by clinicians, and includes advanced MRI investigations alongside genetics and psychometric assessments. 

[Details here.](https://www.epilepsyproject.org.au/)

## An open presurgery MRI dataset of people with epilepsy and focal cortical dysplasia type II

This resource includes data from 85 people affected by epilepsy due to FCD type II and 85 healthy control participants. Imaging modalities include T1w and FLAIR, and the dataset also provides manually labeled regions of interest, and selected clinical features.

[Dataset download.](https://openneuro.org/datasets/ds004199/versions/1.0.5)
[Data descriptor.](https://www.nature.com/articles/s41597-023-02386-7)

## Open multimodal iEEG-fMRI dataset from naturalistic stimulation with a short audiovisual film

The dataset includes intracranial EEG (iEEG) data (51 participants) as well as fMRI recordings (30 participants) during the same task. Eighteen participants performed both iEEG and fMRI versions of the task, non-simultaneously. T1w scans are also provided. The data were acquired during movie watching, and the authors provide detailed speech and video annotations. 

[Dataset download.](https://openneuro.org/datasets/ds003688/versions/1.0.7)
[Data descriptor.](https://www.nature.com/articles/s41597-022-01173-0)

## UNAM dataset

This dataset provides T1w, rs-fMRI and task-fMRI task for evaluation of working memory collected in patients with temporal lobe epilepsy and healthy participants. The dataset also includes cognitive data: Wechsler's Adult Intelligence Scale (WAIS-IV) and Weschsler's Memory Scale (WMS-IV).

[Dataset download.](https://openneuro.org/datasets/ds004469/versions/1.1.3)
[Data descriptor.](https://www.medrxiv.org/content/10.1101/2022.08.02.22278343v2)

## Concurrent deep-brain stimulation and whole-brain functional MRI

This dataset includes data from 26 human patients who underwent electrical stimulation during fMRI (es-fMRI). The patients had medically refractory epilepsy requiring surgically implanted intracranial electrodes in cortical and subcortical locations. One or multiple contacts on these electrodes were stimulated while simultaneously recording BOLD-fMRI activity in a block design. The dataset comprises pre- and post-surgical whole-brain anatomical (T1w) and fMRI imaging at 3T, together with the location of electrodes and the corresponding stimulation protocol parameters.

[Dataset download.](https://openneuro.org/datasets/ds002799/versions/1.0.4)
[Data descriptor.](https://www.nature.com/articles/s41597-020-00595-y)

## Did we miss anything?

Drop a comment below 👇 if you're aware of any other open neuroimaging datasets in epilepsy!
Feel free to drop some key publications associated with these resources 🧠

**Reply by saratheriver** [reply]
Awesome resource! 🔖

**Reply by drzeke** [reply]
Great resources! I know the Human Epilepsy Project will plan to release open datasets in the future too (https://www.humanepilepsyproject.org/research-investigators)

**Reply by new-epilepsy** [reply]
## An open presurgery MRI dataset of people with epilepsy and focal cortical dysplasia type II

This publicly available dataset includes 85 people affected by epilepsy due to FCD type II and 85 healthy control persons. The authors published 3D-T1 and 3D-FLAIR, manually labeled regions of interest, and carefully selected clinical features. The open presurgery MRI dataset may be used to validate existing automated algorithms of FCD detection as well as to create new approaches. Most importantly, it can enable comparability of already existing approaches and support a more widespread use of automated lesion detection tools.

[Dataset download.](https://openneuro.org/datasets/ds004199/versions/1.0.5/download)
[Data descriptor.](https://www.nature.com/articles/s41597-023-02386-7)

**Comment by drzeke** [comment]
### JOB OPPORTUNITIES IN ATLANTA, GA
➡️ **RESEARCH: Post-doctoral fellow.** Come join our research team at Emory University seeking to better understand epilepsy phenotypes and post-surgical outcomes using computational neuroimaging. Our R01 NIH-funded multisite project is now wrapping up data collection after 5 years and offers an unprecedented dataset of > 400 patients with temporal lobe epilepsy who underwent surgery and had seizure outcomes tracked annually. Our cohort of patients and controls features structural, diffusion, and functional imaging with possibilities to further integrate with PET and intracranial EEG. We are looking for a highly motivated applicant who holds a PhD (or is close to completing a PhD) in computational neuroscience, neuroimaging, bioinformatics, or other related fields. Prior experience using neuroimaging processing pipelines and a strong background in Matlab or Python are preferred. Emory offers a great infrastructure for [career development](https://med.emory.edu/education/postdoctoral-training/prospective-postdocs/index.html) for post-docs. Availability as early as May 2024! If interested, email **eze (dot) gleich (at) emory (dot) edu** with CV.

➡️ **CLINICAL: Epilepsy fellow.** We have an opening for a 1-year ACGME Epilepsy fellowship at the Emory University Comprehensive Epilepsy Center starting in July 2024, with possibility to potentially add a second year if desired. This is a busy Level IV center offering a high volume of exposure to ICU EEG, surgical epilepsy, intracranial EEG, and novel applications such as radiofrequency thermocoagulation, RNS (> 55 patients), DBS (> 100 patients for epilepsy), among others. We are looking for a graduating PGY-4 Neurology resident or a recent graduate seeking specialty training in epilepsy and clinical neurophysiology. If interested, email **dwinkel (at) emory (dot) edu** with CV.

**Comment by drzeke** [comment]
## Epilepsy Research Coordinator
### Department of Neurology, Emory University, Gleichgerrcht Lab

### Position Summary:
The Gleichgerrcht Lab within Emory University's Department of Neurology is seeking a dedicated full-time Epilepsy Research Coordinator. This vital role supports the lab’s mission to enhance epilepsy outcomes through advanced research techniques. The coordinator will be responsible for expanding, curating, and maintaining a comprehensive dataset of multimodal neuroimaging and neurophysiological data related to epilepsy. Additionally, the role involves coordinating data collection for innovative tasks in epilepsy, both in the Epilepsy Monitoring Unit and outpatient clinics. This position provides opportunities for skill development, collaboration, and potential leadership in independent studies under the guidance of PI Gleichgerrcht.

This role requires at least a one-year commitment with the possibility of continued support. It is ideal for those pursuing further academic or professional training in PhD programs or clinical tracks in medicine, particularly in neurology, neuroimaging, neurosurgery, or related fields.

### CORE DUTIES:
- Collect and organize raw neuroimaging, neurophysiological, and behavioral data in accordance with IRB guidelines and the lab's database structure.
- Process data using existing pipelines and explore, test, and implement new data processing pipelines.
- Assist with the deployment of new tasks in epilepsy, including screening, recruiting, and obtaining consent from study participants. Conduct virtual and in-person interviews to gather data as needed, schedule testing appointments, distribute experimental tasks, coordinate participant payments, and integrate data into the main dataset.
- Prepare data for input into statistical databases through typing, editing, and organization.

### EDUCATION AND EXPERIENCE:
- A four-year college degree is required.
- Background in neuroimaging, cognitive science, computer science, or statistics is welcome.
- Basic programming skills and experience with neuroimaging data collection are highly valued.

### KNOWLEDGE, SKILLS, AND ABILITIES:
- Proficiency with general computer skills and the ability to quickly learn and master new software.
- Understanding of scientific theory and research methods.
- Ability to meet deadlines with general supervision.
- Strong organizational skills with attention to detail.
- Effective oral and written communication skills.
- Experience working with human study participants.
- Strong programming skills (Python preferred, with some MATLAB and R) are a plus.
- Previous experience with neuroimaging data collection and analysis is advantageous.

### WORKING CONDITIONS:
- The position allows for a hybrid schedule with in-person attendance required for lab meetings and data collection days at both the outpatient clinic (12 Executive Park) and the main hospital (Clifton campus).
- The lab offers access to advanced technology for data processing.
- The PI provides strong mentoring support to help the candidate achieve their academic and professional goals.

### Working for Emory University and living in Atlanta:
Join our expanding lab to contribute to pioneering research at one of the nation’s top medical institutions. Emory University fosters an environment where collaboration, innovation, and discovery thrive. Atlanta offers a vibrant city life with rich cultural heritage, numerous green spaces, and a welcoming community.

- **Career Growth**: Benefit from career development programs, tuition reimbursement, and continuing education opportunities.
- **Supportive Environment**: Enjoy excellent retirement plans, generous time-off, and family care resources.
- **Cultural Enrichment**: Explore museums, trails, and historic sites throughout Atlanta.
- **Convenience**: Take advantage of free commuter programs, ridesharing incentives, discounts, and more.

If this opportunity aligns with your skills and career goals, we encourage you to apply. Thank you for considering employment opportunities with the Department of Neurology at Emory University. 

### **Application Process:**
- The position is available for an immediate start.
- Please reach out to eze(dot)gleich(at)emory(dot)edu with your updated CV.

*Emory University is an equal employment opportunity and affirmative action employer. All qualified applicants will receive consideration for employment without regard to race, color, religion, sex, sexual orientation, gender identity, national origin, disability, protected veteran status, or any other characteristic protected by law.*

**Comment by aliceballerini** [comment]
Hi all!
The __Advanced MRI lab__ at the __University of Modena and Reggio Emilia__ (Italy) is hiring ✨
We are looking for a researcher holding a PhD, or an equivalent qualification, in Engineering, Physics, or Neuroscience with good programming and coding skills and previous expertise in EEG and/or fMRI postprocessing to work on a multicentre study entitled _"Neuro-vascular coupling patterns during wakefulness and sleep in genetic generalized epilepsies: a novel endophenotyping approach for genotype-phenotype correlation"_ 🧠 

If you're interested, search for "Vaudano" in the search box here: https://titulus-unimore.cineca.it/albo/
The application must be filled in and submitted by PICA (https://pica.cineca.it/unimore/) by January 13th 2025, at 1:00 pm (CEST).

**Comment by slic-lab** [comment]
### ILAE School on Neuroimaging 2025
The organizing committee is happy to announce that the ILAE School on Neuroimaging will take place again in 2025!
The ILAE School on Neuroimaging 2025 is divided into two parts:

1. The **Online Lecture Series** of this year’s ILAE School on Neuroimaging will be virtually on Zoom on **7 and 14 March 2025**. Lectures will cover the basics and more advanced topics in neuroimaging in epilepsy.

2. The **Onsite Workshops** of this year’s ILAE School on Neuroimaging will take place from **15 - 17 May 2025** at Seminaris Seehotel Potsdam, Germany. Workshops will give more detailed insight and offer hands-on experience.

Please use our [online registration forms](https://www.ilae.org/congresses/ilae-school-on-neuroimaging-2025) to either register for online attendance or in-person participation onsite in Potsdam.

**Reply by slic-lab** [reply]
Also relevant and interesting: In 2023, the ILAE-YES held a [course on neuroimaging in epilepsy](https://www.youtube.com/watch?v=nUpDvAvdNr4&list=PLTcEDcaqco75NS3DSS7e7IyChxN65ZVCo&ab_channel=ILAEYoungEpilepsySection), primarily intended for clinicians.
